Source/WebCore/svg/SVGAngleValue.cpp

```diff
@@ -125,7 +134,7 @@ ExceptionOr<void> SVGAngleValue::setValueAsString(const String& value)
 
 ExceptionOr<void> SVGAngleValue::newValueSpecifiedUnits(unsigned short unitType, float valueInSpecifiedUnits)
 {
-    if (unitType == SVG_ANGLETYPE_UNKNOWN || unitType > SVG_ANGLETYPE_GRAD)
+    if (unitType == SVG_ANGLETYPE_UNKNOWN || unitType > SVG_ANGLETYPE_TURN)
```

Comment:
I don't think you should have changed this or below. These methods can't know about `SVG_ANGLETYPE_TURN` after all. (These are exposed through IDL.)

Reply:
indeed… damned.
https://svgwg.org/svg2-draft/types.html#InterfaceSVGAngle

Step 2 in https://svgwg.org/svg2-draft/types.html#__svg__SVGAngle__SVG_ANGLETYPE_UNKNOWN

> Otherwise, return [SVG_ANGLETYPE_UNKNOWN](https://svgwg.org/svg2-draft/types.html#__svg__SVGAngle__SVG_ANGLETYPE_UNKNOWN).
> For example, for an [<angle>](https://www.w3.org/TR/css-values/#angles) with a turn unit, [SVG_ANGLETYPE_UNKNOWN](https://svgwg.org/svg2-draft/types.html#__svg__SVGAngle__SVG_ANGLETYPE_UNKNOWN) would be returned.